packages/classic/scss/stepper/_variables.scss

```diff
@@ -82,6 +82,11 @@ $kendo-stepper-progressbar-text: null !default;
 $kendo-stepper-progressbar-fill-bg: null !default;
 $kendo-stepper-progressbar-fill-text: null !default;
 
+/// @deprecated Use `$kendo-stepper-content-transition` instead.
 $kendo-stepper-content-transition-property: height !default;
+/// @deprecated Use `$kendo-stepper-content-transition` instead.
 $kendo-stepper-content-transition-duration: 300ms !default;
```

Comment:
Use var(--kendo-motion-medium-2, .3s) !default; (like in the Default theme)

Reply:
Same point as in the case for Bootstrap, no need to use `--kendo-motion-*`